Client/Ecosia/L10N/String.swift

```diff
@@ -182,5 +182,20 @@ extension String {
         case ecosiaLogoAccessibilityLabel = "Ecosia logo"
         case done = "Done"
         case findInPage = "Find in page"
+        case exportBookmarks = "Export Bookmarks"
+        case importBookmarks = "Import Bookmarks"
+        case exportingBookmarks = "Exporting Bookmarks…"
+        case importingBookmarks = "Importing Bookmarks…"
+        case importedBookmarkFolderName = "Imported Bookmarks (%@)"
+        case bookmarkImportHelpscoutURL = "https://ecosia.helpscoutdocs.com/search?query=Bookmark"
```

Comment:
Does the URL need to stay within the localization `String.swift` file? Trying to get my head around possible string interpolation and utilization within a `UITextView` or `UILabel` maybe. But even so, this URL feels like it may be better into another place? Or even better start to adapt another convenient wrapping `struct` for those things and include the URL in `MultiplyYourImpact:462` 🤷‍♂️ 💭

Reply:
This will be removed, after discussing the matter it turns out this will be a single URL which will adjust to the user's language preference of the browser accordingly.